examples/Autopoint/CMakeLists.txt

```diff
@@ -0,0 +1,165 @@
+# VolEsti (volume computation and sampling library)
```

Comment:
the directory's name should be `autopoint` without capital A, just like the other directories